fk/secretreceive.go

```diff
@@ -130,6 +133,10 @@ func secretReceive() exitCode {
 		}
 	}
 
+	if deletedSecret {
```

Comment:
Currently `if deletedSecret` could be replaced with `if len(decryptedSecrets)`... e.g. it's redundant.

Either `deletedSecrets` should actually mean *deleted secrets **without error*** or it should disappear (variables are bad)